src/resources/packages/classy/style.pcss

```diff
@@ -0,0 +1,62 @@
+:root {
```

Comment:
Pleas make sure that only things that are really about tickets will live here.
Ideally I would like to have the Classy style in Common contain all the presentational classes required to add new components and fields.
Do not be shy about refactoring Common components to have more general-use classes and their styles.

This said: if something is only applicable to a tiny piece of Tickets' component/field, it should live here.